pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -113,7 +113,7 @@ func (s *usmHTTP2Suite) TestRawTraffic() {
 			// The objective of this test is to verify that we accurately perform the parsing of frames within
 			// a single program.
 			messageBuilder: func() []byte {
-				const settingsFramesCount = 100
+				const settingsFramesCount = 220
```

Comment:
why 220?

Reply:
We should test the maximum available
240 - 2 (1 frame for headers, and 1 for data) = 238

Reply:
Since the previous value was 100, I assumed the idea was to have a number large enough, without it being at the limit. So I picked 220 since it is >120 and <240.

Reply:
It depends on the context
in the past we had 2 tests, but now we have a single one, so it should test we support the maximum we intended to (a.k.a filtering and parsing up to 240) frames

Reply:
Consider a mistake on the kernel, where we lower the limit to 230, will the original test captures it?

Reply:
@Yumasi still relevant